### DISCUSSION TITLE
how do we get rid of border

### DISCUSSION THREAD

**Discussion by aDioS786** [epezent/implot#361] [Q&A]
Hi,
I've tried to get rid of this border but it still shows up, tried to set implot style border size to 0 but it still pops up. 
Trying to draw a simple plot:
if (ImPlot::BeginPlot("", NULL, NULL, ImVec2(-1, -1), ImPlotFlags_None, ImPlotAxisFlags_NoGridLines | ImPlotAxisFlags_NoTickMarks | ImPlotAxisFlags_NoTickLabels, ImPlotAxisFlags_NoGridLines | ImPlotAxisFlags_NoTickMarks | ImPlotAxisFlags_NoTickLabels))

But still get this blue border around.

![image](https://user-images.githubusercontent.com/76258953/170974205-f85c5e8c-be6f-4b2b-8489-a9c3b1f09b46.png)

**Comment by epezent** [comment]
```cpp
ImPlot::PushStyleVar(ImPlotStyleVar_PlotPadding, ImVec2(0,0));
if (ImPlot::BeginPlot(...)) {
    ...
    ImPlot::EndPlot();
}
ImPlot::PopStyleVar();
```
![border](https://user-images.githubusercontent.com/29577475/175852469-6a0a1e42-b67b-4b76-adc9-90cd0c7c1926.gif)

**Reply by aDioS786** [reply]
That is great, Thanks. ImPlotStyleVar_PlotPadding did the trick.
